src/style/pauseable_placement.ts

```diff
@@ -97,8 +97,10 @@ class PauseablePlacement {
         const startTime = browser.now();
 
         const shouldPausePlacement = () => {
-            const elapsedTime = browser.now() - startTime;
-            return this._forceFullPlacement ? false : elapsedTime > 2;
+            if (this._forceFullPlacement) {
+                return false;
+            }
+            return browser.now() - startTime > 2;
```

Comment:
I'm not sure what style you all prefer, so curious if you would prefer ternary for conciseness?

```suggestion
            return this._forceFullPlacement ? false : (browser.now() - startTime) > 2;
```